src/App.scss

```diff
@@ -72,6 +72,28 @@ input[class*='inputSingleline'] {
   }
 }
 
+.safetyExit {
```

Comment:
This file seems to be generally for app wide styles. Can you add these styles to `src/components/NavDesktop.js` with the rest of the button's styles?